wolfssl/internal.h

```diff
@@ -1210,7 +1210,7 @@ enum {
             #error "MySQL needs FP_MAX_BITS at least at 16384"
         #endif
 
-        #if WOLFSSL_MAX_RSA_BITS > ENCRYPT_BASE_BITS
+        #if !defined(NO_RSA) && WOLFSSL_MAX_RSA_BITS > ENCRYPT_BASE_BITS
```

Comment:
I think the more direct way to do this is to ensure `defined(WOLFSSL_MAX_RSA_BITS)` rather than `!defined(NO_RSA)`.

Reply:
Agree but neighboring lines that do same check key off `!defined(NO_RSA)`. E.g. this

`       #if !defined(NO_RSA) && WOLFSSL_MAX_RSA_BITS > SP_INT_BITS`

added in recent commit 4b12d2f4ec5a9a5dd0d3d98c95e8b984fa965b76

Reply:
I agree with Hayden. I realize the logic above is wrapped in NO_RSA however that might not always be true. Prefer it gated on the macro it is using.

Reply:
Actually having both is even better

Reply:
I updated this and a few other lines to consistently guard `WOLFSSL_MAX_RSA_BITS` with both `defined(WOLFSSL_MAX_RSA_BITS)` and `!defined(NO_RSA)`.